qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,32 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     * @todo #676:30min Ideally integration test is required to reproduce NPE
+     *  in issue #676.
+     *  To reproduce problem firstly revert changes marked with #676, then
+     *  add to pom.xml of it project at least one exclude rule in
```

Comment:
@vkuchyn *it* should be upper-cased, if you mean integration tests (invoker ones)